R/dependency.R

```diff
@@ -56,6 +56,15 @@ dyPlugin <- function(dygraph, name, path, options = list(), version = "1.0") {
   if (length(options) == 0) {
     options <- JS("{}")
   }
+  # merge options if plugin was already added
```

Comment:
I might be missing something, but doesn't this only pick up new options? (i.e. it wouldn't pick up overwriting of an existing option).